[DOC-539] [2024.2] [EA] Colocated tables with tablespaces

aishwarya24 opened the pull request:
@netlify /preview/explore/colocation/#colocated-tables-with-tablespaces

utkarsh-um-yb commented on docs/content/preview/explore/colocation.md:
````diff
@@ -240,3 +240,133 @@ To set up xCluster for colocated tables, do the following:
+ALTER TABLE ALL IN TABLESPACE tablespace_1 COLOCATED WITH table1 SET TABLESPACE tablespace_2 CASCADE;
+```
+
+This command will move only the tables present in `tablespace_1` that are colocated with `table1`.
````
This command will move only the ~tables~ relations present in `tablespace_1` that are colocated with `table1`

utkarsh-um-yb commented on docs/content/preview/explore/colocation.md:
````diff
@@ -240,3 +240,133 @@ To set up xCluster for colocated tables, do the following:
+
+```
+
+### Backup and restore colocated tables using ysql_dump
````
```suggestion
### Backup and restore colocated Database using ysql_dump
```

aishwarya24 commented:
Created another PR to make sure the content goes to only 2024.2. Addressed the comments in the new one - https://github.com/yugabyte/yugabyte-db/pull/24811
